js.go

```diff
@@ -712,13 +712,20 @@ func (js *js) newAsyncReply() string {
 	}
 	var sb strings.Builder
 	sb.WriteString(js.rpre)
-	rn := js.rr.Int63()
-	var b [aReplyTokensize]byte
-	for i, l := 0, rn; i < len(b); i++ {
-		b[i] = rdigits[l%base]
-		l /= base
+	for {
+		rn := js.rr.Int63()
+		var b [aReplyTokensize]byte
+		for i, l := 0, rn; i < len(b); i++ {
+			b[i] = rdigits[l%base]
+			l /= base
+		}
+		replyID := string(b[:])
+		if _, ok := js.pafs[replyID]; ok {
```

Comment:
I think it would be better to avoid keeping the string and let do Write use the bytes as it was before
```suggestion
		if _, ok := js.pafs[string(b[:])]; ok {
```